ncard_app/templates/tables/organisations.html

```diff
@@ -17,18 +17,19 @@ <h1 class="mb-4">Organisations</h1>
             <th scope="col">Actions</th>
           </tr>
         </thead>
+    <button><a href="{% url 'add-organisations' %}"> Add</a></button>
```

Comment:
The button should be placed to the right of the page header. Follow the example in templates/tables/people.html